modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -182,6 +182,14 @@ export default function ActionBuilder({
 		setValues({conditionExpression});
 	};
 
+	const isValidField = (field: ObjectField) => {
+		return (
+			field.businessType !== 'Aggregation' &&
+			field.businessType !== 'Relationship' &&
+			!field.system
+		);
+	};
```

Comment:
This code can be simplified by using object destructuring:
```suggestion
	const isValidField = ({businessType, system}: ObjectField) => 
		businessType !== 'Aggregation' && businessType !== 'Relationship' && !system);
```